Source/modio/Private/ModioSubsystem.cpp

```diff
@@ -108,6 +108,13 @@ void FModioSubsystem::GetAllMods(const FModioFilterCreator &FilterCreator, const
   QueueAsyncTask( Request );
 }
 
+void FModioSubsystem::GetMod(uint32 ModId, const FModioModDelegate ModDelegate)
+{
+	FModioAsyncRequest_EditMod *Request = new FModioAsyncRequest_EditMod( this, ModDelegate );
```

Comment:
Even though it works, `FModioAsyncRequest_EditMod` should be `FModioAsyncRequest_GetMod`.
I think having a generic AsyncRequest for similar requests like this would be better design. But for now let's just rename it to the `FModioAsyncRequest_GetMod`.

Reply:
Ah right I forgot about this - I tested it with `FModioAsyncRequest_EditMod` before adding `FModioAsyncRequest_GetMod`. I agree that a genery AsyncRequest would be preferable, though.